src/main/java/karuna/karuna_backend/Authentication/DTO/LoginResponseDto.java

```diff
@@ -0,0 +1,15 @@
+package karuna.karuna_backend.Authentication.DTO;
+
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.Date;
+import java.util.List;
+
+@Builder
+@Getter
+public class LoginResponseDto {
+    private String username;
+    //private String jwt;
+    private Date tokenExpirationTime;
```

Comment:
use offsetdatetime, date is in UTC and protected for requests from diffretnt time zones